spec/tapioca/cli/check_shims_spec.rb

```diff
@@ -122,6 +122,21 @@ def baz; end
           refute_success_status(result)
         end
 
+        it "ignores duplicates that have a different superclass" do
```

Comment:
Not sure if this test is correct, since these two cases are defining the same `Foo` class and are, in fact, duplicate definition. For missing superclasses, we should assume `Object`, and then these would be duplicates like expected.